pkg/config/server.go

```diff
@@ -27,8 +27,10 @@ type UserRolesConfig struct {
 }
 
 type UserConfig struct {
-	PublicKey string   `mapstructure:"public_key"`
-	Targets   []string `mapstructure:"targets"`
+	PublicKey   string   `mapstructure:"public_key"`
+	Name        string   `mapstructure:"name"`
+	Description string   `mapstructure:"description"`
```

Comment:
Do you feel strongly about including `description:`?  If not, I'd rather leave it out.  We don't do anything with it (I don't see it being logged, and wouldn't want it to), and server admins can just use comments in the YAML instead.

Reply:
No. It just felt right, since it might encourage better self-documentation, especially if these are eventually managed by an API (PUT calls), but I can't say that I have strong feels for it at the moment. 

Do you want me to go ahead and pull it out for now?

Reply:
Yeah, data that doesn't have a feature driving it bugs me - feels like a software dead-end sign.

Reply:
Oh, it has a feature, remind busy admin who this is and how to get a hold of them, but I get it, I'll pull it for now. :-)